packages/propeller/src/FeedSource.mts

```diff
@@ -101,18 +101,28 @@ export class TailingFeedSource extends EventEmitter {
           wasTail = batch.isTail
         }
       }
+
+      if (!signal.aborted && checkpointError) {
```

Comment:
Should we be checking the aborted case here - the `checkpointError` should only be set in the event it wasn't aborted at the point the error was caught - in which case should we throw it anyway from here (even if someone has aborted between the error being caught, and this logic firing)?

Reply:
Oh sorry - I now see we're setting checkpointError unconditionally - tho should we?